llama2.mojo

```diff
@@ -379,7 +379,7 @@ struct Config:
         var config_data_raw = f.read_bytes(bytes_of_config_params)
         f.close()
         # correct Tensor type and shape for easy reading, without copying data
-        var int32_ptr = config_data_raw._steal_ptr().bitcast[DType.int32]()
+        var int32_ptr = config_data_raw.bitcast[DType.int32]()
```

Comment:
Are you sure we don't still need to steal the pointer also?  `config_data_raw` is going to be deleted after last use which will free this pointer before we are done with it if it is still the owner.

Reply:
I'm not sure

With it remaining the error is 

```
/home/anthony/llama2.mojo/llama2.mojo:382:40: error: 'List[SIMD[si8, 1]]' value has no attribute '_steal_ptr'
        var int32_ptr = config_data_raw._steal_ptr().bitcast[DType.int32]()
```

with it removed the error is
```
/home/anthony/llama2.mojo/llama2.mojo:382:40: error: 'List[SIMD[si8, 1]]' value has no attribute 'bitcast'
        var int32_ptr = config_data_raw.bitcast[DType.int32]()
```

So the theme seems to be `'List[SIMD[si8, 1]]' value has no attribute `

Reply:
I am sure.  😃 The correct thing is to steal the pointer so we take ownership.  The method signature has just changed so it is a bit more elaborate now.  

The full correct line here is:
`var int32_ptr = DTypePointer[DType.int8](config_data_raw.steal_ptr().value).bitcast[DType.int32]()`. 

It can be split into multiple lines if you think it is clearer but the steps are steal the pointer, convert to DTypePointer, and then bitcast to the int32 representation we want.